src/main/kotlin/no/nav/syfo/repository/UtsattOppgaveRepository.kt

```diff
@@ -83,7 +83,7 @@ class UtsattOppgaveRepositoryImp(private val ds: DataSource) : UtsattOppgaveRepo
         }
     }
 
-    override fun opprett(uo: UtsattOppgaveEntitet): UtsattOppgaveEntitet {
+    override fun opprett(innteksmelding: UtsattOppgaveEntitet): UtsattOppgaveEntitet {
```

Comment:
Er egentlig uenig å navngi utsatt oppgave for inntektsmelding. Siden det er en egen tabell for inntektsmelding.
En utsatt oppgave blir feks oppdatert og utsatt flere ganger avhengig av hendelser i vedtaksløsningen.
En inntektsmelding blir ikke oppdatert/endret når den kommer i databasen.

Reply:
OK! Dette var kun fordi super-metoden brukte variabelnavn inntektsmelding, så det ble compile-warning. 
Hvis det er ok, renamer jeg da super-klassens variabelnavn til uo også, så blir det likt.